rust/src/builder.rs

```diff
@@ -67,13 +67,22 @@ pub struct DeltaTableConfig {
     /// Some append-only applications might have no need of tracking any files.
     /// Hence, DeltaTable will be loaded with significant memory reduction.
     pub require_files: bool,
+    /// Controls how many files to buffer from the commit log when updating the table.
+    /// The maximum possible value is 50.
+    /// This defaults to 1
+    ///
+    /// Setting a value greater than 1 results in concurrent calls to the storage api.
+    /// This can decrease latency if there are many files in the log since the 
+    /// last checkpoint.
```

Comment:
It's important to note the tradeoff, otherwise it sounds like we should always just choose the max `log_buffer_size`. Often in these instances, it's a tradeoff between speed and memory usage. Though if you think there's another tradeoff worth mentioning, feel free to add that.

```suggestion
    /// Setting a value greater than 1 results in concurrent calls to the storage api.
    /// This can decrease latency if there are many files in the log since the 
    /// last checkpoint, but will also increase memory usage.
```

Reply:
Agreed and changed this as suggested. The only thing that crossed my mind was if the underlying storage system had some concurrency limits, but I don't think that's a problem at least with the major cloud providers?

Reply:
That's true, rate limits also apply. Feel free to add a comment about that too if you'd like.

Reply:
Added something now: https://github.com/delta-io/delta-rs/pull/1549/files#diff-f81b8b91e9721dc0a3235b6674a57c0700153b92b59f3988d357950a9c8c4760R240
Also updated the docs on the rust side to match the current version